backend/dal/Migrations/v01.02.01/Up/PostUp/01-AdministrativeAreas.sql

```diff
@@ -0,0 +1,655 @@
+PRINT 'Adding municipalities'
```

Comment:
How did you generate all the municipalities for this SQL?

Reply:
I wrote a simple node script that requested all municipality features from the layer. Then extracted the Name, Abbreviation, GroupName from the feature properties fields. 
 `
const axios = require('axios').default;
const shell = require('shelljs');
const fs = require('fs');

const load = () => {
    return axios.get('https://openmaps.gov.bc.ca/geo/pub/WHSE_LEGAL_ADMIN_BOUNDARIES.ABMS_MUNICIPALITIES_SP/wfs?SERVICE=WFS&REQUEST=GetFeature&VERSION=1.3.0&outputFormat=application/json&typeNames=pub:WHSE_LEGAL_ADMIN_BOUNDARIES.ABMS_MUNICIPALITIES_SP')
}

shell.touch('municipalities.sql');

fs.writeFileSync('municipalities.sql', 'PRINT \'Adding municipalities\'\n')
fs.appendFileSync('municipalities.sql', '\n')
fs.appendFileSync('municipalities.sql', 'SET IDENTITY_INSERT dbo.[AdministrativeAreas] ON\n')
fs.appendFileSync('municipalities.sql', '\n')
fs.appendFileSync('municipalities.sql', 'MERGE INTO dbo.[AdministrativeAreas] Target \n')
fs.appendFileSync('municipalities.sql', 'USING ( VALUES \n')

load().then(response => response.data)
    .then(data => {
        const features = data.features;
        console.log(features.length)
        for (let i = 0; i < features.length; i++) {
            const feature = features[i]
            fs.appendFileSync(
                'municipalities.sql',
                `(\n '${feature.properties.ADMIN_AREA_NAME.replace("'", "''")}',\n '${feature.properties.ADMIN_AREA_ABBREVIATION.replace("'", "''")}',\n '${feature.properties.ADMIN_AREA_GROUP_NAME.replace("'", "''")}'\n)${i !== features.length - 1 ? ', ' : ''}`)

        }
    }).then(() => {

        fs.appendFileSync('municipalities.sql', ') AS MSrc (Name, Abbreviation, GroupName)\n')
        fs.appendFileSync('municipalities.sql', 'ON Target.Name = MSrc.NAME\n')
        fs.appendFileSync('municipalities.sql', 'WHEN MATCHED THEN\n')
        fs.appendFileSync('municipalities.sql', 'UPDATE\n')
        fs.appendFileSync('municipalities.sql', 'SET Target.Abbreviation = MSrc.Abbreviation, Target.GroupName = MSrc.GroupName\n')
        fs.appendFileSync('municipalities.sql', 'WHEN NOT MATCHED THEN\n')
        fs.appendFileSync('municipalities.sql', 'INSERT (Name, Abbreviation, GroupName) VALUES (MSrc.Name, MSrc.Abbreviation, MSrc.GroupName);\n')  
    })
`